be/src/olap/in_list_predicate.cpp

```diff
@@ -84,18 +84,51 @@ void CLASS<type>::evaluate(VectorizedRowBatch* batch) const { \
 IN_LIST_PRED_EVALUATE(InListPredicate, !=)
 IN_LIST_PRED_EVALUATE(NotInListPredicate, ==)
 
+#define IN_LIST_PRED_COLUMN_BLOCK_EVALUATE(CLASS, OP) \
+    template<class type> \
+    void CLASS<type>::evaluate(ColumnBlock* block, SelectionVector* selector_vector) const { \
+        if (block->is_nullable()) { \
+            for (int i = 0; i < block->nrows(); ++i) { \
```

Comment:
Any round will iterator all rows even if the block filters many rows.

Reply:
ok, I will optimize here